magento/pwa-studio
# [Doc] Add upgrade steps for scaffolded project

## Description

This PR adds a section in the CHANGELOG on how to upgrade a scaffolded project to the latest version.

## Related Issue





Closes PWA-803

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps


1. Open CHANGELOG.md
2. Verify new **Scaffolder project** section is accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
